dictionary.md

```diff
@@ -0,0 +1,5 @@
+| Short | Description    |
+| ----- |----------------|
+| bgjob | background job |
+| noti | notification   |
+| noqa | NO-QA (NO Quality Assurance) |
```

Comment:
I'm not sure what this file is.  If it's documentation could we have a bit more context and pop it in the `docs` folder?

Reply:
developer always use some short terms and create some new words for variable and function naming.

some developer or some project use `bj`, `bgj` or `bgjob` for Background Job. 
sometime `bj` mean `Backgroup Job`, sometime it mean `Basic Journal`

If developer working on multi project, this file can help developer understand what is `bgj` meaning, or if I need a short term of `Backgroup Job` which one should I use.